packages/wonder-blocks-core/util/enumerate-scroll-ancestors.test.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import {mount} from "enzyme";
+
+import enumerateScrollAncestors from "./enumerate-scroll-ancestors.js";
+
+describe("enumerateScrollAncestors", () => {
+    test("@@iterator() method returns iterator", () => {
+        // Arrange
+        const enumerator = enumerateScrollAncestors((null: any));
+
+        // Act
+        // Flow doesn't want us to index  $FlowFixMe
+        const result = enumerator[Symbol.iterator]();
+
+        // Assert
+        expect(result).not.toBeUndefined();
+        expect(result).toBe(result[Symbol.iterator]());
```

Comment:
This doesn't really test that it's an iterator only that it returns the same value.

Reply:
Yes, but since other tests are testing the iterator part, and then this tests that it matches, then we are in turn, already testing that it returns an iterator.